Jenkinsfile

```diff
@@ -149,7 +149,6 @@ pipeline {
                     steps {
                         script {
                             assert(daosLatestVersion('master', 'el8').matches(/2.7\.\d+.*/))
-                            assert(daosLatestVersion('release/2.4', 'el8').matches(/2.[34]\.\d+.*/))
```

Comment:
This is not fixing the root problem however.  Indeed. it is masking it and sweeping that under the rug.

The root problem (beyond this test failing) is that if somebody were to want to try to do a CI run with a 2.4 release for example, it would fail due to the reasons discussed previously about needing to also query the `*-archive-local` repository if a release is not found in the `*-stable-local` repository.  Indeed, if that problem were fixed, nothing would need doing here.

Will we ever need to test with an archived release?  Who knows.  But since a code change is needed regardless, it would be better to spend the time fixing that than simply eliminating a test failure that is highlighting a gap in our CI ability.